### PR TITLE
Misc. documentation and source comment typos + whitespace fixes 

### LINKED ISSUE #2036: Various documentation and source comment typo fixes
Found via `codespell -q 3 -S *.po,*.pot,./ElectronClient/app/locales,./ReactNativeClient/locales,./CliClient/locales -L adresse,ba,fonction,te,whet`

### PR DESCRIPTION
**Pull request by luzpaz**
Closes #2036

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!--
 
-Please prefix the title with the platform you are targetting:
+Please prefix the title with the platform you are targeting:
```
Comment (tessus) [inline .github/PULL_REQUEST_TEMPLATE]: Laurent uses British English and therefore `targetting` is correct.

Comment (luzpaz) [reply .github/PULL_REQUEST_TEMPLATE]: Sorry, reverted in 7de5f7f

Comment (tessus) [reply .github/PULL_REQUEST_TEMPLATE]: Thanks.

**Comment by tessus** [comment]
Ok, now it depends on Laurent, if he wants to merge it.

LGTM.

**Comment by laurent22** [comment]
Yes that looks good. Some of these changes are on auto-generated files, so they will be overwritten later but that's not a big deal.

**Pull request merged**